app/views/subscriptions/new.html.haml

```diff
@@ -0,0 +1,30 @@
+#subscription_page.container.mx-auto.max-w-xl.py-8.font-normal.leading-normal
+    %h3.text-2xl You grow Your Business
+    %p We grow the next generation of agile talent.
+    %p 
+        Partner with Craft Academy to boost your recruitment process. By purchasing a subscription you'll get access to 
+        full contact information to current learners and alumni.
+    .max-w-sm.mt-4.sm:flex
+        = button_tag 'Yes, I want to purchase a subscription', class: 'ca-button', id: 'subscription-cta'
+
+- content_for :modal_content do
+    = form_with url: subscriptions_path, local: true, method: :post, id: 'payment-form' , class: 'w-full max-w-md' do |form|
+        %div{class: 'flex flex-wrap -mx-3 mb-6'}
```

Comment:
See the comment for `admin/users/new` on how we can rewrite this and move the classed to tailwind.scss

Reply:
Fixed the form. We need to take a look at the text styling.